src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Circle extends Shape implements ShapeBehaviour {
```

Comment:
You don't need to implement the interface if you extend it from the abstract class that implements it